gh-data.py

```diff
@@ -66,6 +66,15 @@ def process_labels(labels):
     }
 
 
+def get_url(text):
+    m = re.search(r"\b(https?://\S+)", text)
+    if m:
+        url = m.group()
+        if url.endswith(','):
+            url = url[:-1]
```

Comment:
Do you need to consider `[](https://example.com)` as well?

Reply:
Yes, good point.